CHANGELOG.md

```diff
@@ -5,6 +5,19 @@
 - The integration testing app formerly under `example/` has been renamed/moved to `test/app/`.
 - The React Native dependency within the integration testing app has been updated from `0.63.4` to `0.66.5`.
 
+### API Changes
+- The `voice.connect` method now has the following function signature
+  ```ts
+  voice.connect(token: string, options?: Voice.ConnectOptions);
+
+  interface Voice.ConnectOptions {
+    contactHandle?: string;
+    params?: Record<string, string>;
+  }
+  ```
+  Not passing an options object or leaving any member of the options object undefined will result in those options using default values.
+  See the API documentation for descriptions of options.
```

Comment:
What are the default values? Maybe add a link to the API docs for the `options`

Reply:
Do we have a permanent link to the docs yet? We don't have the Markdown files on `main` yet so not sure what the best way to link this is.

Reply:
Yes have a permanent link to the docs. It's referenced in the readme
https://github.com/twilio/twilio-voice-react-native/blob/1.0.0-preview.1/docs/voice-react-native-sdk.md
For this PR, just change `1.0.0-preview.1` to `1.0.0-beta.1` so it'll just work once we go beta.